src/deploy-commands.js

```diff
@@ -1,34 +1,61 @@
 const { REST, Routes } = require('discord.js');
-const fs = require('node:fs');
-const path = require('node:path');
 const { DISCORD_SERVER } = require('./config');
 
+const pingCommand = require('./commands/utility/ping');
+const assingMeTaskCommand = require('./commands/utility/assignMeTaskThread');
+const changeStatusCommand = require('./commands/utility/changeStatus');
+const convertTimeCommand = require('./commands/utility/convert-time');
+const myPointsCommand = require('./commands/utility/my-points');
+const prTemplateCommand = require('./commands/utility/pr-template');
+const remindMeCommand = require('./commands/utility/remindme');
+const requestPointCommand = require('./commands/utility/requestPoint');
+const searchCodeCommand = require('./commands/utility/search-code-review');
+const searchMyPointsCommand = require('./commands/utility/search-my-points');
+const sendMessageCommand = require('./commands/utility/send-message');
+const serverCommand = require('./commands/utility/server');
+const userCommand = require('./commands/utility/user');
+const votePointsCommand = require('./commands/utility/vote-points');
+
 module.exports = (client) => {
```

Comment:
NIT: I think module.exports should be the last declaration in the file, it's weird that it's the first declaration